packages/icon-button/src/IconButton.story.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { css } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { select, boolean } from '@storybook/addon-knobs';
+import IconButton, { Variant } from './IconButton';
+
+const background = css`
+  width: 60px;
+  height: 60px;
+  background-color: ${uiColors.gray.dark3};
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+storiesOf('IconButton', module)
+  .add('Default', () => (
+    <IconButton
```

Comment:
I was under the impression `IconButton` would shadow the available Icons that we have.

Also I just realized....the `glyph` prop on the `Icon` component  doesn't contain the list of available glyphs. I.e. it'd be better if we had an array of the available strings as the value for the glyph prop then we could autocomplete nicely.

And then this component would shadow that.

Reply:
Hmm... We would need to update the `createIconComponent` method to accept an optional `Glyphs` type parameter that defaults to `string` for that. Can totally do that, but it doesn't yet support this.

Reply:
AFAIK (<- haha love using those) `<Icon />` does support this autocomplete